package ch.cern.todo.core.application.exception;

public class NotFoundException extends RuntimeException {}
